omod/src/main/webapp/fragments/patientdashboard/contactInfoInline.gsp

```diff
@@ -4,12 +4,12 @@
 
 <div class="contact-info-inline">
     <span>
-        ${ ui.format(config.patient.personAddress).replace("\n", ", ")}
+        ${ ui.encodeHtmlContent(ui.format(config.patient.personAddress).replace("\n", ", "))}
         <em>${ ui.message("coreapps.person.address")}</em>
     </span>
     <span class="left-margin">
         <span id="coreapps-telephoneNumber">
-            ${config.patient.telephoneNumber ?: ''}
+            ${ ui.encodeHtmlContent(config.patient.telephoneNumber ?: '') }
```

Comment:
Is this xss that appears in the patient registration summary page, as shown in the attached image?

![screen shot 2016-12-04 at 11 27 24 am](https://cloud.githubusercontent.com/assets/7016326/20867518/cd26fb86-ba14-11e6-804f-f77022b580bc.png)

If so, I had to fix that issue with changes to uicommons. I can submit a pull request for those changes shortly.